assets/scripts/book/api.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+const url = "https://wdi-library-api.herokuapp.com/books"
```

Comment:
Question: Do we have to require a file from the config and store, where we will set this url, or is this a shortcut that works, as well?